cmd/minikube/cmd/config/enable.go

```diff
@@ -52,6 +53,21 @@ var addonsEnableCmd = &cobra.Command{
 		if addon == "olm" {
 			out.Styled(style.Warning, "The OLM addon has stopped working, for more details visit: https://github.com/operator-framework/operator-lifecycle-manager/issues/2534")
 		}
+		addonBundle, ok := assets.Addons[addon]
+		if ok {
+			maintainer := addonBundle.Maintainer
+			if maintainer == "Google" || maintainer == "Kubernetes" {
+				out.Styled(style.Tip, `{{.addon}} is an addon maintained by {{.maintainer}}. For any concerns contact minikube on GitHub.
+You can view the list of minikube maintainers at: https://github.com/kubernetes/minikube/blob/master/OWNERS`,
+					out.V{"addon": addon, "maintainer": maintainer})
+			} else if verifiedMaintainer := addonBundle.VerifiedMaintainer; verifiedMaintainer != "" {
+				out.Styled(style.Tip, `{{.addon}} is maintained by {{.maintainer}} for any concerns contact {{.verifiedMaintainer}} on GitHub.`,
+					out.V{"addon": addon, "maintainer": maintainer, "verifiedMaintainer": verifiedMaintainer})
```

Comment:
```suggestion
			} else if addonBundle.VerifiedMaintainer != "" {
				out.Styled(style.Tip, `{{.addon}} is maintained by {{.maintainer}} for any concerns contact {{.verifiedMaintainer}} on GitHub.`,
					out.V{"addon": addon, "maintainer": maintainer, "verifiedMaintainer": addonBundle.VerifiedMaintainer})
```

The assigning within the if statement is a bit long and makes it a bit unreadable, I feel doing a direct comparison without assigning a new variable would be better